docs/advanced_usage.md

```diff
@@ -221,6 +222,28 @@ the same flag:
 $ notary remove example/collections delegation/path/target --roles=targets/releases
 ```
 
+## Recovering a delegation
+
+It is possible for delegations to get into a state where they delegation file is not
+signed by any currently valid keys. Even when a new valid key is added to the delegation
+role, clients will refuse to pull the existing file due to signature verification failure.
+However the existence of the delegation file will cause the server to reject a new delegation
+file set to version `0`. 
+
+The new `notary witness` command adds signatures to an existing invalid delegation file to
```

Comment:
Good point on "new".

The current implementation actually always completely resigns with a version and expiry bump. Any further intelligence is in the future.